vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -124,15 +134,27 @@ run_stress_test(
 )
 ```
 
+**NOTE:** One important input argument that affects the behavior of where the
+results are written, is the argument `return_results`. Per default it is set to
+`FALSE`, which means that results are written as csv files to a sub directory in
+the `output_path` and the user must open these files to view the results.
+If the user wants to view the results directly in RStudio, they can set the
+argument to `TRUE` which will return them as data frames in the R session. This
+means they are not written to the designated `output_path` however, so the user
```

Comment:
if themselves then users

Reply:
I wanted to use a gender neutral they here. Does this imply I should use plural? I thought this worked different..

Reply:
ahh, i thought I deleted this comment, your formulation is correct afaik